### ISSUE TITLE
Unable to import files having .pcss extension

### ISSUE THREAD

**Issue by asifshirazi** [jonathantneal/postcss-partial-import#30]
After updating from **2.1.0** to **3.0.1** I've to manually add "extension" on every files in my stylesheet, otherwise partial-import can't locate my files.

> Here is my config

```
require("postcss-partial-import")({ extension: ".pcss" }),
```

> My old way to import

```
@import "inc/reset";
@import "inc/variables";
@import "inc/mixins";
```

> And now I've to add "extension"

```
@import "inc/reset.pcss";
@import "inc/variables.pcss";
@import "inc/mixins.pcss";
```

**Comment by jonathantneal** [comment]
Thank you. This is now resolved in [v3.1.0](https://github.com/jonathantneal/postcss-partial-import/commit/f21afdbec6328a4fb174d08e1b0e04a1fb1605ef), which has been published.

**Comment by Nabeelhassan** [comment]
v3.1.0 is still not working on windows (10) x64

```
Error in plugin 'gulp-postcss'
Message:
    Failed to find 'inc/variables'
    in [
        E:\local\webapp\private\src\postcss
    ]
Details:
    fileName: E:\local\webapp\private\src\postcss\test-web.pcss
Stack:
Error: Failed to find 'inc/variables'
    in [
        E:\local\webapp\private\src\postcss
    ]
    at E:\local\node_modules\postcss-import\lib\resolve-id.js:61:11
[17:43:29] Finished 'compile:postcss:local' after 251 ms
```
